tortoise/fields.py

```diff
@@ -89,6 +89,7 @@ def __init__(
         self.model = model
         self.reference = reference
         self.description = description
+        super().__init__(**kwargs)  # type: ignore # mypy issue 4335
```

Comment:
This is causing issues, so I'm going to revert it. When removing it, and using your example I can't actually get it to not call the entire super chain.

Reply:
@grigi The problem is with the Python's method resolution order. In my example VirtualField's `__init__` will never called. 

I thought my solution could be a problem, when no multiple inheritence was in use, but it worked for me and all tests were passed.

Now I read a lot about MRO and found that, this behavior is because python first go through the  1st base class's parent chain, then the 2nd, if there are no common classes in there. In my example only common class were `object`. So the solution was to extend my `VirtualField` class from `Field` and magically __init__ is called and everything works as expected.

So you could remove that commented line. Sorry about that...

Reply:
Could you please do a documentation pr describing the caveat & process?
Maybe update against develop, as there is a lot of doc changes lined up for the next major release already

Reply:
sure, next week